### PR TITLE
User-profile card should open on down arrow, close opener gap

### PR DESCRIPTION
**Pull request by annabelsegalld2l**
Just bundled 2 fixes together here:
1. The card should open on the down arrow. I realized that the dropdown mixin does not listen for the down key since it's not commonly used as an opener, so I added a listener here.
2. The visual-diffs here show the same issue that is happening in consistent-eval: profile images were rednering too tall, causing a too-large opener gap. This can be fixed by ensuring the profile-image has a display: block. I did double-check that this would not affect CE by running their visual-diffs as well.

### CODE REVIEW

File: user-profile-card-styles.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ export const profileCardStyles = css`
 	grid-template-rows: [start] 1rem [basic-info-start] 6rem [illustration-end] auto [basic-info-end awards-start] auto [awards-end contact-start] auto [contact-end end];
 }
 
+/* Opener */
+.d2l-labs-user-profile-card-opener {
+	/* Use block to ensure profile-image always renders
+	with correct height (can be taller when using image) */
+	display: block;
```
Comment (annabelsegalld2l) [inline user-profile-card-styles.js:20]: See visual diffs: random gap is gone 🎉

Comment (svanherk) [reply user-profile-card-styles.js:20]: Does the actual `d2l-profile-image` have two different heights depending on whether it's image or initials?  Or only when we wrap it in a dropdown?

**Comment by svanherk** [comment]
> The card should open on the down arrow. I realized that the dropdown mixin does not listen for the down key since it's not commonly used as an opener

Should it?  I'm good with what you have here, but maybe check with @geurts he's not expecting this to work for all dropdown openers

File: user-profile-card-styles.js (hunk 1 above)
Comment (annabelsegalld2l) [reply user-profile-card-styles.js:20]: The profile image is used in quite a few spots so I was hesitant to go in and change it, as some of the other components may be relying on that behaviour.

Comment (svanherk) [reply user-profile-card-styles.js:20]: We can tackle that in the eventual migration to `core`, seems to be related to `line-height`

**Review by svanherk — APPROVED** [review]
Visual diffs look much better!

**Pull request merged**